frontend/src/components/Credentials/GOrphanedCredentialIcon.vue

```diff
@@ -0,0 +1,26 @@
+<!--
+SPDX-FileCopyrightText: 2023 SAP SE or an SAP affiliate company and Gardener contributors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+<!--
+SPDX-FileCopyrightText: 2023 SAP SE or an SAP affiliate company and Gardener contributors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+<template>
+  <v-tooltip location="top">
+    <template #activator="{ props: activatorProps }">
+      <v-icon
+        v-bind="activatorProps"
+        icon="mdi-alert-circle-outline"
+        end
+        size="small"
+        color="warning"
+      />
+    </template>
+    Associated credential does not exist
```

Comment:
Distinguish between Secret and WorkloadIdentity in the text. The binding should be available via inject, right?
Then also include <namespace>/<name> in the text.

Reply:
binding available in parent component, can be passed as prop. Added information to tooltip